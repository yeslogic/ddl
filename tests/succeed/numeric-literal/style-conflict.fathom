//~ test-normalisation = true

u32_add "nope" 0xcafe
