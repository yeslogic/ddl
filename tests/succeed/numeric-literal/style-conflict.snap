stdout = '''
("nope" : U32) + (0xcafe : U32) : U32
'''
stderr = ''
